app/views/inputs/multipleselect.slim

```diff
@@ -1,4 +1,4 @@
 input type="hidden" name="#{name}[]"
 select name="#{name}[]" multiple="multiple"
 	- options.each do |ovalue, display|
-		option value=ovalue selected=("selected" if value.to_s.include?(ovalue)) = display
+		option value=ovalue selected=("selected" if value.map(&:to_s).include?(ovalue.to_s)) = display
```

Comment:
I'm going to refactor this to pass in the options with selected already set, that way I can more easily test it.